gm4_guidebook/generate_guidebooks.py

```diff
@@ -34,6 +34,31 @@
 # TODO:
 # merge some functions to reduce fuction call overhead
 
+DEFAULT_COLORS = {
+  "minecraft:leather_boots": 10511680,
+  "minecraft:leather_chestplate": 10511680,
+  "minecraft:leather_helmet": 10511680,
+  "minecraft:leather_leggings": 10511680
+}
+
+DEFAULT_OVERLAY_COLORS = {
+  "empty": 0
+}
+
+IS_DYED = [
+  "minecraft:leather_boots",
+  "minecraft:leather_chestplate",
+  "minecraft:leather_helmet",
+  "minecraft:leather_leggings"
```

Comment:
Should also included `leather_horse_armor` and `wolf_armor` for completeness?

Reply:
Done. I've unified the definition, so the tint checker uses this list instead of another inline list